src/javascript/app/ng-wig/ng-wig-toolbar.provider.js

```diff
@@ -11,7 +11,7 @@ angular.module('ngWig').provider('ngWigToolbar', function () {
   var defaultButtonsList = ['list1', 'list2', 'bold', 'italic', 'link'];
 
   var isButtonActive = function () {
-    return this.command && document.queryCommandState(this.command);
+    return this.command !== undefined && document.queryCommandState(this.command);
```

Comment:
Do you mean to replace it for the whole function or just the first part of the return statement?